Source/WebCore/layout/formattingContexts/inline/InlineContentConstrainer.cpp

```diff
@@ -733,6 +733,11 @@ Vector<size_t> InlineContentConstrainer::computeBreakOpportunities(InlineItemRan
     return breakOpportunities;
 }
 
+LayoutUnit InlineContentConstrainer::computeLineWidthFromSlidingWidth(InlineLayoutUnit indentWidth, SlidingWidth slidingWidth) const
+{
+    return LayoutUnit::fromFloatCeil(indentWidth + slidingWidth.width() + LayoutUnit::epsilon());
+}
```

Comment:
nit: this looks like just a static helper (but I don't feel too strongly about it).